src/main.rs

```diff
@@ -192,6 +208,46 @@ struct CheckRelease {
     )]
     release_type: Option<ReleaseType>,
 
+    /// Use only the default and explicitly added features.
+    #[arg(
+        long,
+        help_heading = "Features",
+        conflicts_with = "only_explicit_features"
+    )]
+    default_features: bool,
+
+    /// Use no features except the explicitly mentioned ones.
```

Comment:
```suggestion
    /// Use no features except ones explicitly added by other flags.
    ///
    /// Using this flag disables the heuristic that enables all features
    /// except `unstable`, `nightly`, `bench`, `no_std`, and ones starting with `__`.
```
Repetition in docs isn't bad, so long as it's brief, informative, and something the user needs to know but wouldn't have found out otherwise.